ixwebsocket/IXWebSocketTransport.cpp

```diff
@@ -730,7 +779,13 @@ namespace ix
     WebSocketSendInfo WebSocketTransport::sendPing(const std::string& message)
     {
         bool compress = false;
-        return sendData(wsheader_type::PING, message, compress);
+
+        WebSocketSendInfo info = sendData(wsheader_type::PING, message, compress);
+
+        std::lock_guard<std::mutex> lck(_lastSendPingTimePointMutex);
+        _lastSendPingTimePoint = std::chrono::steady_clock::now();
```

Comment:
I think we should save `_lastSendPingTimePoint` only if send was successful

Reply:
Agree